third_party/flatbuffers/flatbuffers.BUILD

```diff
@@ -88,7 +88,7 @@ cc_binary(
         ":platform_freebsd": [
             "-lm",
         ],
-        ":windows": [],
+        ":windows": ["/DEFAULTLIB:msvcrt.lib"],
```

Comment:
What is the link error?  This change seems odd.

Reply:
Looking at TF 2.12: https://github.com/tensorflow/tensorflow/blob/8e2b6655c0c488290179ab90a0daed0f6d3006f7/third_party/flatbuffers/flatbuffers.BUILD#L90

There is no explicit adding of this linkopt, so I'm not sure where the difference is.  This doesn't seem GPU-specific, and our current MSVC tests are passing in non-gpu builds.

Reply:
The linking error happened everytime if without `/DEFAULTLIB:msvcrt.lib`, even I completely reinstall Win10. I have changed it as comment.